python/foglamp/services/south/server.py

```diff
@@ -58,6 +59,8 @@ class Server(FoglampMicroservice):
 
     _task_main = None
 
+    _restart_config = None
```

Comment:
we already have plugin['restart'] = yes | no and return handle with new config why do we need this now again?